tests/test-runner/bin/Makefile.am

```diff
@@ -2,3 +2,14 @@ pkgdatadir = $(datadir)/@PACKAGE@/test-runner/bin
 dist_pkgdata_SCRIPTS = \
 	test-runner.py \
 	zts-report.py
+#
+# These scripts are compatibile with both Python 2.6 and 3.4.  As such the
+# python 3 shebang can be replaced at install time when targeting a python
+# 2 system.  This allows us to maintain a single version of the source.
+#
+#if HAVE_PYTHON_2
+#install-exec-hook:
+#	sed --in-place 's/^#!\/usr\/bin\/python3/#!\/usr\/bin\/python/' \
+#	    $(DESTDIR)$(pkgdatadir)/test-runner.py \
+#	    $(DESTDIR)$(pkgdatadir)/zts-report.py
+#endif
```

Comment:
Since this is all commented out, can we just remove it?

Reply:
The comments here were for testing and not meant to be included.  We actually need this for a minimal CentOS 6/7 installs.